pcsx2/gui/Panels/MemoryCardListPanel.cpp

```diff
@@ -1030,9 +1026,8 @@ void Panels::MemoryCardListPanel_Simple::OnOpenItemContextMenu(wxListEvent& evt)
 		} else {
 			junk->Append(McdMenuId_Create, _("&Create a new card ..."));
 		}
-		else
-			junk->Append( McdMenuId_Create, _("Create a new card ...") );
-
+	} else {
+		junk->Append(McdMenuId_Create, _("&Create a new card ..."));
```

Comment:
These changes belong to the menu shortcut commit (which currently doesn't compile).

Reply:
I move it back(this appearing in the wrong commit was due to git messing with the PR). Seems like the commit itself was fine, as that was the only change I made and it just finished compiling succesfully.